### DISCUSSION TITLE
Import and decrypt encrypted text passages from enex-file

### DISCUSSION THREAD

**Discussion by MichaelHeydukFraba** [akosbalasko/yarle#544] [Ideas]
I have lots of passwords in my Evernote .enex file. All are locally encrypted passwords with the Evernote encrypt-Text-Feature.
It would be great if I could specify a local password in the Import Settings, so that the enex-importer decrypts all encrypted texts.
Maybe this feature would help lots of other people too?

**Comment by akosbalasko** [comment]
Hi @MichaelHeydukFraba ,
It is implemented in https://github.com/akosbalasko/yarle/issues/543 and already released.
